### PR TITLE
8306881: Update FreeType to 2.13.0

### PR DESCRIPTION
**Pull request by alisenchung**
update freetype version to 2.13, all tests passed

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8306881](https://bugs.openjdk.org/browse/JDK-8306881): Update FreeType to 2.13.0

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13841/head:pull/13841` \
`$ git checkout pull/13841`

Update a local copy of the PR: \
`$ git checkout pull/13841` \
`$ git pull https://git.openjdk.org/jdk.git pull/13841/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13841`

View PR using the GUI difftool: \
`$ git pr show -t 13841`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13841.diff">https://git.openjdk.org/jdk/pull/13841.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13841#issuecomment-1536786151)

### CODE REVIEW

File: src/java.desktop/share/native/libfreetype/include/freetype/config/ftmodule.h [hunk 1]
```diff
@@ -15,19 +15,19 @@ FT_USE_MODULE( FT_Driver_ClassRec, tt_driver_class )
 FT_USE_MODULE( FT_Driver_ClassRec, t1_driver_class )
 FT_USE_MODULE( FT_Driver_ClassRec, cff_driver_class )
 FT_USE_MODULE( FT_Driver_ClassRec, t1cid_driver_class )
-// FT_USE_MODULE( FT_Driver_ClassRec, pfr_driver_class )
-// FT_USE_MODULE( FT_Driver_ClassRec, t42_driver_class )
-// FT_USE_MODULE( FT_Driver_ClassRec, winfnt_driver_class )
-// FT_USE_MODULE( FT_Driver_ClassRec, pcf_driver_class )
-// FT_USE_MODULE( FT_Driver_ClassRec, bdf_driver_class )
+//FT_USE_MODULE( FT_Driver_ClassRec, pfr_driver_class )
```
Comment (prrace) [inline src/java.desktop/share/native/libfreetype/include/freetype/config/ftmodule.h:18]: if you add back the spaces the diff for this file should go away.

**Review by prrace — COMMENTED** [review]
Can you also confirm that you compiled this using the option discussed in UPDATING.txt, ie
---
When updating specify --with-freetype=bundled to test builds to expose build issues.
This is important because presently on Linux the build defaults to linking against the
system library and does not attempt to compile the sources.
---

File: src/java.desktop/share/native/libfreetype/include/freetype/internal/ftmmtypes.h [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ * David Turner, Robert Wilhelm, Werner Lemberg, George Williams, and
+ * Dominik Röttsches.
+ *
+ * This file is part of the FreeType project, and may only be used,
+ * modified, and distributed under the terms of the FreeType project
+ * license, LICENSE.TXT.  By continuing to use, modify, or distribute
+ * this file you indicate that you have read the license and
+ * understand and accept it fully.
+ *
+ */
```
Comment (DamonGuy) [inline src/java.desktop/share/native/libfreetype/include/freetype/internal/ftmmtypes.h:18]: Is this the only new file being added?

**Review by DamonGuy — COMMENTED** [review]
md file overall looks fine to me so far

**Comment by alisenchung** [comment]
I built jdk with option --with-freetype=bundled on macosx, linux, and windows without issues

File: src/java.desktop/share/native/libfreetype/include/freetype/internal/ftmmtypes.h (hunk 2 above)
Comment (alisenchung) [reply src/java.desktop/share/native/libfreetype/include/freetype/internal/ftmmtypes.h:18]: yea

**Review by prrace — APPROVED** [review]
"I built jdk with option --with-freetype=bundled on macosx, linux, and windows without issues"

OK - approving.

**Pull request closed without merging**